include/mbedtls/check_config.h

```diff
@@ -66,6 +66,65 @@
 #error "MBEDTLS_HAVE_TIME_DATE without MBEDTLS_HAVE_TIME does not make sense"
 #endif
 
+/* Check that each MBEDTLS_ECP_DP_xxx symbol has its PSA_WANT_ECC_xxx counterpart,
+ * but only when config_psa.h is evaluated. */
+#if defined(MBEDTLS_PSA_CRYPTO_CONFIG) || defined(MBEDTLS_PSA_CRYPTO_C)
```

Comment:
I think this should be just `PSA_CRYPTO_C`. We want the property "PSA curves is a superset of ECP curves" to be true in all builds (that have PSA).

Reply:
I took these guards from `build_info.h` [line 105](https://github.com/Mbed-TLS/mbedtls/blob/5e678fd4d2f530e8082096b25d5ac916fb378e56/include/mbedtls/build_info.h#L105)

Question: having `MBEDTLS_PSA_CRYPTO_CONFIG` or `MBEDTLS_PSA_CRYPTO_C` isn't a superset of just `MBEDTLS_PSA_CRYPTO_C`? So the target:

>We want the property "PSA curves is a superset of ECP curves" to be true in all builds (that have PSA)

should be achieved, ins't it?